components/post_view/combined_system_message/combined_system_message.tsx

```diff
@@ -320,7 +320,7 @@ export class CombinedSystemMessage extends React.PureComponent<Props> {
         );
     }
 
-    renderMessage(postType: string, userIds: string[], actorId: string): JSX.Element {
+    renderMessage(postType: string, userIds: string[], actorId?: string): JSX.Element {
         return (
             <React.Fragment key={postType + actorId}>
```

Comment:
The use of `undefined` here in the concatenation remains unresolved by this PR, but technically works since we just concatenate the word `undefined` to get a unique case.